src/cljs/kixi/nhs/exploration_ui/data.cljs

```diff
@@ -0,0 +1,56 @@
+(ns kixi.nhs.exploration-ui.data
+  (:require [om.core :as om :include-macros true]
+            [ajax.core :refer (GET POST)]
+            [clojure.walk :as walk]
+            [clojure.string :as str]))
+
+(defn date-parser [date]
+  (let [date-str (-> date (.match #"\d{4}"))]
+    (new js/Date date-str)))
+
+(defn get-url
+  "Build url string by joining site url with params."
+  [& api-method]
+  (let [hostname "http://54.154.11.196/api/3/action/"]
```

Comment:
I think a function for now as it shoud be the host that we are accessing anyway when it is a CKAN plugin. (ckan-host) or similar